src/services/ImagesService/ImagesService.test.ts

```diff
@@ -24,65 +22,27 @@ describe('ImagesService', () => {
     })
   })
 
-  describe('fetchImages', () => {
-    const mockImages = [
-      {
-        markedAsInvalid: 0,
-        masks: [],
-        name: 'image_0.jpg',
-        skipped: 0,
-        labellers: ['1'],
-        createdOn: new Date('June 13, 2022, 12:00:00'),
-        isCompleted: false,
-        sampleLocation: '1',
-        sampleReference: '1',
-      },
-      {
-        markedAsInvalid: 0,
-        masks: [],
-        name: 'image_0.jpg',
-        skipped: 0,
-        labellers: ['1', '2'],
-        createdOn: new Date('June 01, 2022, 12:00:00'),
-        isCompleted: false,
-        sampleLocation: '1',
-        sampleReference: '1',
-      },
-    ]
-
-    const databaseSpy = jest.spyOn(DatabaseService, 'getDocuments')
-
-    beforeEach(() => {
-      databaseSpy.mockReturnValue(Promise.resolve(mockImages))
-    })
-
-    it('should query from images collection and filter by isCompleted false and order by createdOn', async () => {
-      const queryConstraints: QueryConstraint[] = [where(IS_COMPLETED, '==', false), orderBy(CREATED_ON)]
-      await fetchImages('1111')
-      expect(DatabaseService.getDocuments).toHaveBeenCalledWith('images', queryConstraints)
-    })
-
-    it('should filter images by user id 2', async () => {
-      const result = await fetchImages('2')
-      const filteredImages = [
-        {
-          markedAsInvalid: 0,
-          masks: [],
-          name: 'image_0.jpg',
-          skipped: 0,
-          labellers: ['1'],
-          createdOn: new Date('June 13, 2022, 12:00:00'),
-          isCompleted: false,
-          sampleLocation: '1',
-          sampleReference: '1',
-        },
-      ]
-      expect(result).toEqual(filteredImages)
-    })
-
-    it('should filter images by user id 1', async () => {
-      const result = await fetchImages('1')
-      expect(result).toEqual([])
+  describe('fetchImageToLabel', () => {
+    const mockImage: Image = {
+      id: '1',
+      name: 'image_1.jpg',
+      sampleLocation: '1',
+      sampleReference: '1',
+      masks: [],
+      labellers: [],
+      markedAsInvalid: 0,
+      isCompleted: false,
+      createdOn: new Date(),
+    }
+
+    it('should call the fetchImageToLabel cloud function and return the image ', async () => {
```

Comment:
I am missing the tests when there is no image returned

Reply:
Added